rslib/backend.proto

```diff
@@ -1052,6 +1054,19 @@ message FindAndReplaceIn {
   string field_name = 6;
 }
 
+message BrowserColumns {
+  repeated BrowserColumn columns = 1;
+}
+
+message BrowserColumn {
```

Comment:
If this message is only going to be used in the BrowserColumns message, I'd embed it there and call it "Column"

Reply:
That's what I started with but I didn't like the sound of `BrowserColumns.Column`. Would you mind if I added a `BrowserTable` message (just as a namespace) and made Column, Row etc. members?

Reply:
Sounds even better

Reply:
I've tried it, but apparently I can only use top-level messages in an rpc definiton.
`rpc AllBrowserCardColumns(Empty) returns (BrowserTable.Columns);` results in `AttributeError: module 'anki._backend.backend_pb2' has no attribute 'Columns'`. 😕

Reply:
Ah,  I guess that makes sense. BrowserColumns.Column works for me - it's not much different to having a browser_columns.rs that has a Column struct in it.